css/styles.css

```diff
@@ -107,21 +113,17 @@
 }
 
 .home {
-    padding: 66px 30px 31px 30px;
+    padding-top: 78px;
+    padding-bottom: 95px;
 }
 
-.home-image3 {
-    padding-top: 23%;
-}
-
-.home-image1,
-.home-image4 {
-    padding-top: 214px;
+.services {
+    padding-top: 40px;
 }
 
 .service-items {
-    margin: 25px;
-    width: 28%;
+    padding-bottom: 41px;
+    width: 27%;
```

Comment:
@Chem-e I don't think you should use width percentages. We are already using it via bootstrap cols.